jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -3,13 +3,12 @@ lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
 pipeline {
     options {
         timeout(time: 4, unit: 'HOURS')
-        copyArtifactPermission("${JOB_BASE_NAME}")
-        buildDiscarder(logRotator(artifactNumToKeepStr: '1'))
     }
     agent none
     environment {
         AGENT_X64 = 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
         AGENT_ARM64 = 'Jenkins-Agent-al2-arm64-c6g4xlarge-Docker-Host'
+        IMAGE_RPM = 'opensearchstaging/ci-runner:ci-runner-rockylinux8-opensearch-dashboards-integtest-v2' // required for rpm to create digest sha256 correctly with rpm 4.12+
```

Comment:
The image name has "integtest" in it but now it can be used for assembling. We may later need to either make its name more generic or create a new one name for assemble.

Reply:
We can, however, build another `build.rockylinux` image just for the purpose of this.
That can be a future improvement so dont get blocked now.
Thanks.